### ISSUE TITLE
Documentation de "une possibilité"

### ISSUE THREAD

**Issue by laem** [betagouv/mon-entreprise#1310]
Je vois que ce mécanisme qui n'en n'est pas un est marqué comme "va évoluer bientôt" dans #1293 
Mais en attendant, j'ai l'impression qu'il est nulle part dans la doc, or c'est une brique assez importante du langage. 

Notamment le fait de devoir spécifier une valeur par défaut de ce style me semble vraiment pas évident.

```yaml
par défaut: "'tulipe'"
```
Je pense qu'il faut l'ajouter, qu'en pensez-vous ?

**Comment by laem** [comment]
Pas très beau 😅 en tout cas ça liste notre utilisation chez ecolab de "une possibilité".

**Comment by johangirod** [comment]
Effectivement... Mais depuis que `par défaut` est évalué et peut référencer une autre règle il fallait différencier le cas de la chaîne de caractère.

Pour la doc, j'avoue être un peu embêté, je ne sais pas trop quoi dire. La logique d'évaluation du mécanisme est pour l'instant vide : https://github.com/betagouv/mon-entreprise/blob/master/publicodes/source/mecanisms/one-possibility.ts

Toute la logique derrière est dans `RuleInput.ts`, d'où le fait que la doc publicodes core reste vide pour l'instant. Lorsque une possibilité codera pour un type Enum, on aura plus de chose à dire. Ça devrait venir bientôt.

**Comment by laem** [comment]
OK je crois voir le souci. Mais j'ai l'impression qu'on peut quand même utiliser ce mécanisme dans publicodes sans génération de simulateur.

Par exemple, pour [ce petit bout de code](https://publi.codes/studio/transport/voiture/empreinte?code=transport%3A%0Atransport . voiture%3A%0Atransport . voiture . motorisation%3A%0A%20 question%3A Quel type de voiture utilisez-vous %3F%0A%20 description%3A Si vous utilisez plusieurs voitures (par exemple dans le cas où vous n'en possédez pas une)%2C choisissez la réponse la plus représen%20 tative de votre usage.%0A%20 par défaut%3A "'thermique'"%0A%20 formule%3A%0A%20%20%20 une possibilité%3A%0A%20%20%20%20%20 choix obligatoire%3A oui%0A%20%20%20%20%20 possibilités%3A%0A%20%20%20%20%20%20%20 - thermique%0A%20%20%20%20%20%20%20 - hybride%0A%20%20%20%20%20%20%20 - électrique%0Atransport . voiture . motorisation . thermique%3A%0Atransport . voiture . motorisation . électrique%3A%0Atransport . voiture . motorisation . hybride%3A %0A%0Atransport . voiture . empreinte%3A%0A%20 formule%3A %0A%20%20%20 variations%3A %0A%20%20%20%20%20 - si%3A motorisation %3D 'thermique'%0A%20%20%20%20%20%20%20 alors%3A 666%0A%20%20%20%20%20 - si%3A motorisation %3D 'hybride'%0A%20%20%20%20%20%20%20 alors%3A 555%0A%20%20%20%20%20 - sinon%3A 444).

Là, dans le studio, on n'en fera rien parce qu'on ne peut pas spécifier à la fois la formule "une possibilité" (qui en l'absence de la génération du formulaire ne sert qu'à typer les valeurs possibles de "motorisation") et une valeur de simulation, par exemple "hybride". 

Mais dans l'utilisation en vrai de publicodes, même si c'est une coquille vide, le mécanisme permet de générer une documentation du typage de "motorisation".
